src/main/resources/rankups.yml

```diff
@@ -5,40 +5,50 @@
 # Join the discord server for live support: https://discord.gg/maB4382 (buyers only)
 #
 
-# this name doesn't matter
-Aexample:
-  # the name of the group
-  # players have to be in this rank to rankup
-  rank: 'A'
-  # the name of the rank a player can rankup to
-  next: 'B'
-  # List of requirements to go to the next rank
-  # This example will charge 1000 money to rankup from A to B.
-  # https://github.com/okx-code/Rankup3/wiki/List-of-Requirements
-  # custom requirements can also be added by other plugins.
-  requirements:
-    - 'money 1000'
-  # the console will run these commands when a player ranks up
-  # nb: groups are automatically changed with vault
-  #commands:
-  # this will run when a player ranks up from A to B.
-  #- 'say {PLAYER} well done for ranking up from {OLD_RANK} to {RANK}!'
-Bexample:
+ExampleA: # This name doesn't matter, it can be anything you want!
+  rank: 'A' # This is the rank where you start from. Use the displayname if you have it!
+  next: 'B' # This is the rank where you go next. Read it as "From A to B"
+  requirements:  # Here go your requirements. You can have as many as you want.
+# List of requirements: https://github.com/okx-code/Rankup3/wiki/List-of-Requirements
+    - 'money 5000'
+    - 'xp-level 5'
+
+ExampleB:
   rank: 'B'
   next: 'C'
   requirements:
-    - 'money 2500'
-Cexample:
+    - 'money 10000'
+    - 'xp-level 10'
+  rankup: # requirements-not-met is a custom message that gets displayed when you DON'T meet the requirements.
+    requirements-not-met: "&3You need &d&n$10K &3and &d&n10 XP levels&3!" # It supports (RGB if 1.16+) color codes.
+
+ExampleC:
   rank: 'C'
   next: 'D'
   requirements:
-    - 'money 5000'
-    - 'xp-level 2'
-  # you can have a custom messages for each rank
-  # the paths of these messages are the same as in the messages for your locale
+    - 'money 15000'
+    - 'xp-level 12'
+    - 'total-mob-kills 15'
+  rankup: # Introduction to placeholders and multi-line messages!
+    requirements-not-met: |- # These 2 characters indicate to Rankup that it's a multi-line message.

```

Comment:
"# The characters '|-' indicate the following indented likes are a multi-line string; literally the message. No need for '\n' escape characters when using |-. See https://github.com/okx-code/Rankup3/wiki/FAQ#how-do-i-write-multi-line-messages"